guides/cpp-intro.md

```diff
@@ -121,6 +116,35 @@ myled = 1;
 Here the `=` operator is overloaded and is used to set the value of the led.
 So when you change the value of `myled` here you are actually setting the value of the led.
 
+# Printing in mbed c++
+
+In normal c++ one would use `prinft("Hello world\n")` or `cout << "Hello world" << endl` to print strings to the standard output of the program.
```

Comment:
In C++ it's common to use `cout`, however in mebd it is `logger.printf`. if we just want this to be for the mbed-programmer eaching `cout` is not that useful